ui/ui-components/__tests__/components/managedCard.tsx

```diff
@@ -0,0 +1,11 @@
+import { render } from "@testing-library/react";
+import ManagedCard from "../../components/lib/ManagedCard";
+
+describe("managedCard", () => {
+  it("will render a spinner when in the loading state", async () => {
+    let card = render(<ManagedCard title="test" loading={false}></ManagedCard>);
+    expect(card.container).toMatchSnapshot("noLoading");
+    card = render(<ManagedCard title="test" loading={true}></ManagedCard>);
+    expect(card.container).toMatchSnapshot("loading");
```

Comment:
This is great!
Minor note, you don't need to name snapshots if you don't want to, jest will handle it 👍